### ISSUE TITLE
Notification action brings qbittorrent out of tray

### ISSUE THREAD

**Issue by 7ff04da4aa6ac37197e249299066ac30** [ErikReider/SwayNotificationCenter#132]
I'm not sure if the issue I'm experiencing is at all related to swaync, but I'd like to ask here just in case. I'll otherwise ask for help at qbittorrent or firefox/librewolf if it turns out this isn't a swaync issue.

I'm on Arch Linux, using sway (obviously) and waybar to display swaync. I haven't messed with the default keybindings, as I'm satisfied with them. Anyway, whenever librewolf issues a notification (such as when using discord or taking a screenshot with its built-in utility), it shows up as a swaync notification. So far, so good. But when I click on the notification with the left mouse button — therefore issuing the default action —, qbittorrent is pulled into the top of the display. The action otherwise works fine (for example, on discord in the browser, clicking has the effect of jumping to the specific chat and message that caused the notification). Ideally, I would like qbittorrent not to appear when clicking an unrelated notification.

This only happens when qbittorrent is minimized to the waybar tray. If it is already "open", this behaviour does not occur. Also, when I send the open qbittorrent window to the scratchpad, mimicking the minimized to tray feature, the issue does not occur. This has been my "fix" for now, though I'd much rather have it sit normally on the tray and only be invoked when actually relevant.

I don't know which logs to attach here, if any. Anyway, sorry if this isn't the place to ask for help, but I'm at a loss here.

**Comment by ErikReider** [comment]
It sounds like a qbittorrent issue to me... Closing

**Comment by 7ff04da4aa6ac37197e249299066ac30** [comment]
Actually, after digging around a bit, it has to do with Qt6 and gnome/gtk support

qBittorrent/qBittorrent#17165
qBittorrent/qBittorrent#17054

Just posting this here in case someone else stumbles upon this.

**Comment by ErikReider** [comment]
In what browser? Chrome, Firefox?

**Comment by ErikReider** [comment]
I can actually reproduce this with Discord notifications in Firefox

**Comment by 7ff04da4aa6ac37197e249299066ac30** [comment]
Yes, that seems to be the case. In case someone has the same issue in the near future (i.e. before qbittorrent gets this fixed), the current workaround is to use a version of qbittorrent compiled with Qt5, not Qt6.
